ui/src/app/applications/components/application-operation-state/application-operation-state.tsx

```diff
@@ -93,7 +93,15 @@ export const ApplicationOperationState: React.StatelessComponent<Props> = ({appl
         });
     }
     if (operationState.syncResult) {
-        operationAttributes.push({title: 'REVISION', value: <Revision repoUrl={utils.getAppDefaultSource(application).repoURL} revision={operationState.syncResult.revision} />});
+        operationAttributes.push({
+            title: 'REVISION',
+            value: (
+                <Revision
+                    repoUrl={utils.getAppDefaultSource(application).repoURL}
+                    revision={utils.getAppDefaultSyncRevision(application) + utils.getAppDefaultSyncRevisionExtra(application)}
```

Comment:
Check if the extra part should be part of the revision field value, which might be included as part of the hyperlink

Reply:
Just to follow up on this, the link is 'broken' because the 'and (1) more' phrase is included

<img width="654" alt="revisionLink" src="https://github.com/argoproj/argo-cd/assets/7726022/93949ab9-5562-4d1b-baf3-f0ee45c791dc">